### DISCUSSION TITLE
Revamp - intro quest for perks

### DISCUSSION THREAD

**Discussion by Mithreas** [zunath/SWLOR_NWN#1470] [Ideas]
[Mith] Perks on creation are still a bit clunky.  Would it be possible to open the Perk window automatically when a new character logs in, along with a server message telling them how to open it again?
[Z] Can we bake that into a tutorial quest or something?
[Mith] In the creation room?  I guess!

Basically - something to guide newbie players on how to access their Perk menu so that they can actually equip something.

**Comment by zunath** [comment]
I'm not sure this is really needed after all. People don't seem to be too confused about the skills/perk system. Going to convert this to a discussion item for now.

**Comment by JohnWildkins** [comment]
The NPE's not too bad (having just recently gone through it as a new player!) at least until you get to Veles. Still, a little help might be nice.

**Comment by Cavcode** [comment]
The new highlight functionality could be pretty useful for that. Akin to your standard nwn tutorial these days that walk you step by step with pop-up instructions
